corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -63,6 +69,83 @@ def _delete_doc_if_exists(self, doc_id):
             self.elasticsearch.delete(self.index_info.index, self.index_info.type, doc_id)
 
 
+class BulkElasticProcessor(ElasticProcessor):
```

Comment:
Seems very similar to the `BulkPillowReindexProcessor` class.

Reply:
Agreed. I think at some point we can consolidate these processors, but right now reindex accessors don't actually try to handle errors from getting the docs or from inserting them into ES. It's not clear to me where or if those errors fit into the reindex workflow.

I thought it might make sense to write this out here specifically for cases, ensure it works/helps us scale and then try to generalize it to do the same for xforms/case search/whatever else